### ISSUE TITLE
Could not open static file ''

### ISSUE THREAD

**Issue by w2naf** [jupyter/notebook#225]
Hi.  I have Ubuntu 14.04 and used to have a working Jupyter notebook system.  I went to upgrade and found that I now need to install Jupyter notebook manually.  I followed the README.md instructions for Trusty Tar exactly:

```
sudo apt-get install nodejs-legacy npm python-virtualenv python-dev
python2 -m virtualenv ~/.virtualenvs/notebook
source ~/.virtualenvs/notebook/bin/activate
pip install --upgrade setuptools pip
git clone https://github.com/jupyter/notebook.git
cd notebook
pip install -r requirements.txt -e .
jupyter notebook
```
I needed to update and install some dependencies along the way, but I eventually got it to install.  It now runs, but it can't find the CSS files.  I've also tried this on my Ubuntu 12.04 machine, with the same result.  Does anyone know a solution?

```
(notebook)w2naf@w2naf-VirtualBox:~/test$ jupyter notebook
[I 00:39:13.502 NotebookApp] Serving notebooks from local directory: /home/w2naf/test
[I 00:39:13.502 NotebookApp] 0 active kernels 
[I 00:39:13.502 NotebookApp] The IPython Notebook is running at: http://localhost:8888/
[I 00:39:13.502 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
libGL error: pci id for fd 14: 80ee:beef, driver (null)
Created new window in existing browser session.
[E 00:39:13.978 NotebookApp] Could not open static file ''
[W 00:39:14.171 NotebookApp] 404 GET /static/components/jquery-ui/themes/smoothness/jquery-ui.min.css (127.0.0.1) 14.64ms referer=http://localhost:8888/tree
[W 00:39:14.175 NotebookApp] 404 GET /static/style/style.min.css (127.0.0.1) 2.63ms referer=http://localhost:8888/tree
[W 00:39:14.185 NotebookApp] 404 GET /static/components/es6-promise/promise.min.js (127.0.0.1) 2.04ms referer=http://localhost:8888/tree
[W 00:39:14.188 NotebookApp] 404 GET /static/components/requirejs/require.js (127.0.0.1) 2.10ms referer=http://localhost:8888/tree
[W 00:39:14.191 NotebookApp] 404 GET /static/tree/js/main.min.js (127.0.0.1) 1.87ms referer=http://localhost:8888/tree
```

Thanks.

**Comment by minrk** [comment]
Hi, we haven't yet released 4.0 of the notebook, and there are still kinks to work out for running from git master. I would recommend getting the lates stable version of The notebook, which is IPython 3.2.1.

**Comment by w2naf** [comment]
Thanks.  I removed all of my ipython installation, and then put it back in using `sudo pip install ipython`.  That brought me back to IPython 3.2.1, as well as a stable version of Jupyter notebook.  It all works now.

Thanks for your work and good luck with you development,
Nathaniel
